yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -715,17 +716,16 @@ Internet-Draft    Updated YANG Module Revision Handling    February 2024
 
       A particular revision of an imported module adheres to an import's
       "recommended-min-date" extension statement if the imported
-      module's latest revision date satisfies, i.e., does not precede,
-      the revision date argument of the "recommended-min-date" extension
-      statement in the importing module.
+      module's latest revision date satisfies, i.e., does not precede
+      (in time), the revision date argument of the "recommended-min-
```

Comment:
I'll be modifying this as per suggestion from @jsterne

Reply:
Ack okay.  My concern here was to avoid someone interpreting precede as a previous entry in the history rather than a date based comparison.

Reply:
It says revision date and later, so I think that should be clear enough.